build.gradle

```diff
@@ -63,8 +64,16 @@ dependencies {
 
   signature 'org.codehaus.mojo.signature:java17:1.0@signature'
 
-  testCompile 'org.spockframework:spock-core:0.7-groovy-1.8'
+  testFixturesImplementation 'org.jenkins-ci.main:jenkins-test-harness:2.44'
+
   jenkinsTest 'org.jenkins-ci.main:jenkins-test-harness:2.44@jar'
+
+  testCompile 'org.spockframework:spock-core:0.7-groovy-1.8'
+
+  jenkinsNextTestCompile "org.jenkins-ci.main:jenkins-war:${nextJenkinsVersion}"
```

Comment:
Should this better be `jenkinsNextTestImplementation`?
Otherwise you are mixing `...Compile` and `...Implementation` in this file, which is inconsistent.